Parse/Internal/PFDevice.m

```diff
@@ -27,14 +27,19 @@
     const char *charName = [name UTF8String];
 
     size_t size;
-    sysctlbyname(charName, NULL, &size, NULL, 0);
+    if (sysctlbyname(charName, NULL, &size, NULL, 0) < 0) {
```

Comment:
This should be `== 0`, not `< 0`. Theoretically an implementation could return `1` to indicate error as well.